client/knowledge_base_search/src/containers/ViewAllContainer/styles.scss

```diff
@@ -10,6 +10,28 @@
   padding: 2rem;
   box-shadow: -2px 0 1px 0 rgba(0,0,0,0.3);
   z-index: 1;
+
+  button.view_all_question--button,
+  button.close_view_all--button {
+    padding: 0;
+    border: none;
+  }
+
+  button.view_all_question--button {
+    margin: 0.5rem 0;
+    width: 100%;
+    font-weight: 500;
+    font-size: 0.8rem;
+    color: #008571;
+    text-align: left;
+  }
+
+  button.view_all_question--button:hover,
+  button.view_all_question--button:focus,
+  button.close_view_all--button:hover,
+  button.close_view_all--button:focus {
+    text-decoration: underline;
+  }
```

Comment:
Since you're using the scss pre-processor now, can you do

```css
button {
  &.view_all_questions--button,
  &.close_view_all--button {
    // styles
  }

  etc...
}
```


Actually, if you refactor the naming so you have 

`view_all_questions--button` and `view_all_questions--button--close`

you could start doing patterns like

```css
.view_all_question--button {
  // base styles

  &--close {
    // modifier styles
  }
}
```